azext_iot/operations/digitaltwin.py

```diff
@@ -0,0 +1,227 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Unpublished works.
+# --------------------------------------------------------------------------------------------
+
+from os.path import exists
+from knack.util import CLIError
+from azure.cli.core.util import read_file_content
+from azext_iot._constants import PNP_ENDPOINT
+from azext_iot._factory import _bind_sdk
+from azext_iot.common.shared import SdkType
+from azext_iot.common._azure import get_iot_hub_connection_string
+from azext_iot.common.utility import (shell_safe_json_parse,
+                                      unpack_msrest_error)
+from azext_iot.operations.pnp import (iot_pnp_interface_show,
+                                      iot_pnp_interface_list)
+from azext_iot.operations.hub import iot_hub_monitor_events
+
+
+DEFAULT_INTERFACE_KEY_NAME = 'urn_azureiot_ModelDiscovery_DigitalTwin'
```

Comment:
nit: If the interface key isn't configurable just call it `INTERFACE_KEY_NAME`.

Reply:
Done